ci/gpu/build.sh

```diff
@@ -63,10 +63,8 @@ gpuci_mamba_retry install "cudatoolkit=$CUDA_REL" \
 # `event_loop_policy`. See https://github.com/dask/distributed/pull/4212 .
 gpuci_mamba_retry install "pytest-asyncio=<0.14.0"
 
-# https://docs.rapids.ai/maintainers/depmgmt/
-# gpuci_mamba_retry remove -f rapids-build-env
-# gpuci_mamba_retry install "your-pkg=1.0.0"
-
+# Install cucim in order to test GDS spilling
+gpuci_mamba_retry install "cucim"
```

Comment:
Yes, `cucim` should be able to fallback to regular POSIX read/write.

Reply:
Is this `cucim` installation temporary? Or permanent? If it's permanent, we should probably move it to the `gpuci_mamba_retry install` lines above that include the other RAPIDS libraries. We shouldn't remove the `# https://docs.rapids.ai/maintainers/depmgmt/` comment block here. It's for testing dependencies as outlined here https://docs.rapids.ai/maintainers/depmgmt/#testing-a-dependency

Reply:
This should be temporary.  Though hopefully at some point soon we should have an alternate GDS Python solution

Reply:
@ajschmidt8, I moved the installation of `cucim` to the line above. We expect this to be temporary but it might take some time before we have an alternate package. Also, reverted the comment-out lines.

Reply:
Looks good to me. Thanks @madsbk for addressing this. As soon as @ajschmidt8 approves we can get this merged.